instrumentation/opentelemetry-instrumentation-confluent-kafka/src/opentelemetry/instrumentation/confluent_kafka/__init__.py

```diff
@@ -178,9 +183,11 @@ def commit(self, *args, **kwargs):
         return self._consumer.commit(*args, **kwargs)
 
     def consume(
-        self, num_messages=1, *args, **kwargs
+        self, *args, **kwargs
     ):  # pylint: disable=keyword-arg-before-vararg
```

Comment:
After the change, is the pylint directive still needed?

Reply:
No, it's not. Removed!